### DISCUSSION TITLE
Arm flatpak

### DISCUSSION THREAD

**Discussion by Jaymz665** [organicmaps/organicmaps#4294] [Q&A] [answered]
I use a smartphone on linux (PmOs) because I have a big request to add an arm64 (aarch64) version to flatpack.
Thank you in advance

**Comment by biodranik** [comment]
CC @Ferenc-

**Comment by Ferenc-** [comment, marked as answer]
1. There is a flatpak for `aarch64` [already](https://github.com/organicmaps/organicmaps/issues/3547#issuecomment-1270364114).
2. FYI [PmOS does not recommend using flatpaks](https://wiki.postmarketos.org/wiki/Flatpak).
